webrtc/RTCConfiguration-iceServers.html

```diff
@@ -0,0 +1,570 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCConfiguration iceServers</title>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<script src='RTCConfiguration-helper.js'></script>
+<script>
+  'use strict';
+
+  // Test is based on the following editor's draft:
+  // https://w3c.github.io/webrtc-pc/archives/20170605/webrtc.html
+
+  // The following helper function is called from
+  // RTCConfiguration-helper.js:
+  //   config_test
+
+  /*
+    4.3.2.  Interface Definition
+      [Constructor(optional RTCConfiguration configuration)]
+      interface RTCPeerConnection : EventTarget {
+        ...
+      };
+
+    4.2.1.  RTCConfiguration Dictionary
+      dictionary RTCConfiguration {
+        sequence<RTCIceServer>   iceServers;
+        ...
+      };
+
+    4.2.2.  RTCIceCredentialType Enum
+      enum RTCIceCredentialType {
+        "password",
+        "oauth"
+      };
+
+    4.2.3.  RTCOAuthCredential Dictionary
+      dictionary RTCOAuthCredential {
+        required DOMString macKey;
+        required DOMString accessToken;
+      };
+
+    4.2.4.  RTCIceServer Dictionary
+      dictionary RTCIceServer {
+        required (DOMString or sequence<DOMString>) urls;
+                 DOMString                          username;
+                 (DOMString or RTCOAuthCredential)  credential;
+                 RTCIceCredentialType               credentialType = "password";
+      };
+   */
+
+  test(() => {
+    const pc = new RTCPeerConnection();
+    assert_equals(pc.getConfiguration().iceServers, undefined);
+  }, 'new RTCPeerConnection() should have default configuration.iceServers of []');
```

Comment:
This: 

> 'new RTCPeerConnection() should have default configuration.iceServers of []'

Does not reflect the test: 

> `assert_equals(pc.getConfiguration().iceServers, undefined);`

If `iceServers` is an empty array, it would not equal `undefined`